### ISSUE TITLE
bug: No mapping found for macro recording "q"

### ISSUE THREAD

**Issue by zaynOm** [LazyVim/LazyVim#4606]
### Did you check docs and existing issues?

- [x] I have read all the LazyVim docs
- [x] I have updated the plugin to the latest version before submitting this issue
- [x] I have searched the existing issues of LazyVim
- [x] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.10.1

### Operating system/version

ArchLinux

### Describe the bug

The default macro recording key map 'q' does not work, when I run `:verbose nmap q` it says `No mapping found`

### Steps To Reproduce

just click 'q' to start a recording

### Expected Behavior

start a new recording

### Repro

vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system("curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua"))()

require("lazy.minit").repro({
  spec = {
    { "LazyVim/LazyVim", import = "lazyvim.plugins" },
    -- add any other plugins here
  },
})

**Comment by dpetka2001** [comment]
It does work. `nmap` doesn't return anything because it probably isn't defined in some Neovim Lua API, but comes from vim's default bindings. Just press `qq` to start a recording in register `q` and you will see the message right bottom `recording @q`.

**Comment by zaynOm** [comment]
yeah you're right `nmap` does not show the keymap, but I can assure you it does not work, when I click `qq` it does not show `recording @q`

**Comment by dpetka2001** [comment]
There's something wrong with either your configuration or some conflicting mapping with the terminal you're using. 

Following a screencast with default LazyVim installation from scratch

https://github.com/user-attachments/assets/ad9601a6-54fb-42c9-80ce-75c9ac6477da

Try with a default LazyVim installation like shown in the screencast.

**Comment by zaynOm** [comment]
I did the test and it worked just fine, I don't know what's causing the issue, It was working just fine, here is [my config](https://github.com/zaynOm/lazyvim-config).

**Comment by dpetka2001** [comment]
I tried your dotfiles. Your problem is in the file `/lua/plugins/lualine.lua`. You're using a values function for `opts` in which you return a new table. That will override LazyVim's lualine settings. That's why you don't see the message for the macros. But if you continue and register a macro, it does work. It's just that you don't have visual confirmation in your lualine and you don't see it. Delete your lualine file and you will see the default behavior.

**Comment by zaynOm** [comment]
I see, I appreciate the help
